pages/linux/brightnessctl.md

```diff
@@ -0,0 +1,24 @@
+# brightnessctl
+
+> Utility for reading and controlling device brightness for GNU/Linux operating systems.
+> More information: <https://github.com/Hummer12007/brightnessctl>.
+
+- List devices with available brightness control:
+
+`brightnessctl -l`
+
+- Get the currently set brightness of the device:
+
+`brightnessctl get`
+
+- Set brightness to certain percent of the maximum (eg. 50%):
+
+`brightnessctl set {{50%}}`
+
+- Increase brightness by certain percent (eg. 10%):
+
+`brightnessctl set {{+10%}}`
+
+- Decrease brightness by certain percent (eg. 10%):
+
+`brightnessctl set {{-10%}}`
```

Comment:
What do you think about this?
```suggestion
- Increase or decrease the brightness of the display backlight by a certain percentage:

`brightnessctl set {{+|-10%}}`
```

Reply:
I'm not sure, splitting the `+` and `-` is easier to read; and this page has 5 examples. :ok_hand: